apps/frontend/src/router.tsx

```diff
@@ -33,7 +33,7 @@ const RewardsPage = loadable(
   () => import('./app/5_pages/RewardsPage/RewardsPage'),
 );
 
-export const router = createBrowserRouter([
+export const router = createHashRouter([
```

Comment:
Please add additional environment variable to flag build as an IPFS and only use `createHashRouter` if it exists, otherwise - keep `createBrowserRouter` as a router